File: MobilePhone/src/mobilephone/MobilePhone.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package mobilephone;

/**
 *
 * @author Blake
 */
public class MobilePhone {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Smartphone mobilePhone1 = new Smartphone("Samsung", "Galaxy s7 Edge",
                5.5, 769.99);
        Smartphone mobilePhone2 = new Smartphone("Iphone", "5c", 4.0, 149.99);
        System.out.println(mobilePhone1.getPhone());
        System.out.println(mobilePhone2.getPhone());
        mobilePhone1.updatePrice(709.99);
        System.out.println(mobilePhone1.getPhone());
        
        FlipPhone mobilePhone3 = new FlipPhone(true, "key pad");
        FlipPhone mobilePhone4 = new FlipPhone(false, "keyboard");
        System.out.println(mobilePhone3.getPhone());
        System.out.println(mobilePhone4.getPhone());
    }
    
}

File: mobilePhone.cpp
#include "SmartPhone.h"
#include "flipPhone.h"

int main() {
	smartPhone mobilePhone1("Samsung", "Galaxy s7 Edge", 5.5, 769.99);
	smartPhone mobilePhone2("Iphone", "5c", 4.0, 149.99);
	mobilePhone1.getPhone();
	mobilePhone2.getPhone();

	flipPhone mobilePhone3("Tmobile", "Key Pad", true, 50.00);
	flipPhone mobilePhone4("Verizon", "Keyboard", false, 20.00);
	mobilePhone3.getPhone();
	mobilePhone4.getPhone();
}